.env.example

```diff
@@ -0,0 +1,22 @@
+# Logging
+DEBUG=info*,error*,warn*,debug*
+# keys are derived from the mnemonic above
```

Comment:
The above comment doesn't make sense without the mnemonic being included in this file, I think its safe to delete